tools/cli/adminKafkaCommands.go

```diff
@@ -432,13 +429,8 @@ func deserializeMessages(messages [][]byte, skipErrors bool) ([]*replicator.Repl
 	return replicationTasks, skipped
 }
 
-func decode(message []byte, val *replicator.ReplicationTask) error {
```

Comment:
I am pretty sure this is wrong, but I don't have enough context and can't figure out what happens here.